my-dsl.md

```diff
@@ -2,37 +2,46 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+The name of the language is CHEM. It is fairly old, so it does not have a webpage. However, a paper that talks about it can be found at this link: http://dl.acm.org/citation.cfm?id=315691
 
 # Domain
 _Describe the language's domain in five words._
 
+Creates chemical structure diagrams.
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+When a program runs, it reads lines and draws something onto the canvas based on what was on the line. It can draw things like benzene and can connect them with things like double bonds. The final output is a chemical structure diagram.
 
```

Comment:
An example diagram (for those unfamiliar with the domain) would help make your description a little bit clearer.